parse-references.js

```diff
@@ -22,15 +22,26 @@ var extractionRules = {
             normative: "#normative-references > dl",
             informative: "#informative-references > dl"
         }
-    }
+    },
+    anolis: {
+        generator: "Anolis",
+        sectionId: {
+            normative: "anolis-references"
+        },
+        listSelector: {
+            normative: "#anolis-references > dl"
+        }
+    },
 };
 
 function extract(url, cb) {
     jsdom.env({
         url: url,
         features: {
             FetchExternalResources: ['script'],
-            ProcessExternalResources: ['script']
+            ProcessExternalResources: ['script'],
+            // only load respec, to avoid jsdom bugs
+            SkipExternalResources: /^((?!respec).)*$/
```

Comment:
indeed; in general, we probably need a core spec-util module to DRY these modules.